src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/TseAdmCloseServiceTest.java

```diff
@@ -84,12 +78,11 @@ void updateStatusToClose() {
             .isNull();
     }
 
-    @ParameterizedTest
+    @Test
     @MethodSource("generateCloseApplicationMarkdown")
-    void generateCloseApplicationMarkdown(boolean appHasDoc, boolean appHasDetails,
-                                          boolean hasDoc, boolean hasAdditionalInfo) {
+    void generateCloseApplicationMarkdown() {
         GenericTseApplicationType tseApplicationType =
-            getTseAppType(appHasDoc, appHasDetails, hasDoc, hasAdditionalInfo);
+            getTseAppType(true, true, true, true);
```

Comment:
This can be refactored. The params to this method are useless now, they don't do anything (intelliJ even says so). Can also remove the @'MethodSource tag